### DISCUSSION TITLE
Audio Compression & Speaker Diarization

### DISCUSSION THREAD

**Discussion by thedpod** [Kav-K/GPTDiscord#262] [Ideas]
Hey, big fan of the tool, it's really cool. I have an idea regarding audio file transcriptions, it would be great if the audio file was compressed as whisper api only allows 25MB maximum. Also would be cool to have speaker diarization in the transcription, meaning the transcription would distinguish between speakers, something like https://github.com/m-bain/whisperX 

e.g. 
Speaker 1: Hello
Speaker 2: Hi
Speaker 1: How are you?
etc.

Thoughts?

**Comment by ghost** [comment]
Le jeu. 13 avr. 2023 à 03:15, thedpod ***@***.***> a écrit :

> Hey, big fan of the tool, it's really cool. I have an idea regarding audio
> file transcriptions, it would be great if the audio file was compressed as
> whisper api only allows 25MB maximum. Also would be cool to have speaker
> diarization in the transcription, meaning if the transcription could
> distinguish between speakers, something like
> https://github.com/m-bain/whisperX
>
> Thoughts?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Kav-K/GPT3Discord/discussions/262>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZYJR5CY4AR3U5XQVYPQ62TXA6RZHANCNFSM6AAAAAAW4VDTKE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
Very good idea. I think it could make the use of GPT better.👌👍
